### PR TITLE
release script

### PR DESCRIPTION
**Pull request by elharo**
@chanseokoh @etanshaul

### CODE REVIEW

File: scripts/prepare_release.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+# Usage: IncrementVersion <version>
+IncrementVersion() {
+    local version=$1
+    local minorVersion=$(echo $version | sed 's/[0-9][0-9]*\.[0-9][0-9]*\.\([0-9][0-9]\)*/\1/')
+    local nextMinorVersion=$((minorVersion+1))
+    echo $version | sed "s/\([0-9][0-9]*\.[0-9][0-9]*\)\.[0-9][0-9]*/\1.$nextMinorVersion/"
+}
+
+[ $# -ne 2 ] || DieUsage
```
Comment (chanseokoh) [inline scripts/prepare_release.sh]: nit: double `[[` for consistency?

File: scripts/prepare_release.sh [hunk 2]
```diff
@@ -0,0 +1,77 @@
+#!/bin/bash -
+# Usage: ./prepare_release.sh <release version>
+
+set -e
+
+Colorize() {
```
Comment (chanseokoh) [inline scripts/prepare_release.sh]: Not used.

Comment (elharo) [reply scripts/prepare_release.sh]: done

File: scripts/prepare_release.sh (hunk 1 above)
Comment (elharo) [reply scripts/prepare_release.sh]: done

**Pull request merged**